python/plugins/processing/script/ScriptTemplate.py

```diff
@@ -95,13 +103,14 @@ def initAlgorithm(self, config: Optional[dict[str, Any]] = None):
 
         # We add the input vector features source. It can have any kind of
         # geometry.
-        self.addParameter(
-            QgsProcessingParameterFeatureSource(
-                self.INPUT,
-                "Input layer",
-                [QgsProcessing.SourceType.TypeVectorAnyGeometry],
-            )
+
+        input_layer=QgsProcessingParameterFeatureSource(
```

Comment:
A space before and after the `=` is missing:
`input_layer = QgsProcessingParameterFeatureSource(`